Formula/terminally-tetris.rb

```diff
@@ -0,0 +1,24 @@
+class TerminallyTetris < Formula
+  desc "Play Tetris in the Terminal"
```

Comment:
given this is a university Software Systems final project, is it notable enough / widely-used enough to be in homebrew? (rather than hosted in a personal tap) Also, will it be actively maintained going forward?